package-res/resources/web/pentaho/data/filter/And.js

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "./AbstractTreeFilter",
+  "require",
+  "./Or",
+], function(AbstractTreeFilter, require, Or) {
+  "use strict";
+
+  /**
+   * @name And
+   * @memberOf pentaho.data.filter
+   * @class
+   * @extends pentaho.data.filter.AbstractTreeFilter
+   * @amd pentaho/data/filter/And
+   *
+   * @classdesc The `And` class implements a type of AbstractTreeFilter {@link pentaho.data.filter.AbstractTreeFilter}.
```

Comment:
`@classdesc` The `And` class implements the intersection of a list of filters, each of which defines a set.